lib/browser.ts

```diff
@@ -851,11 +851,11 @@ export class ProtractorBrowser extends AbstractExtendedWebDriver {
    * browser.get('https://angularjs.org/');
    * expect(browser.getCurrentUrl()).toBe('https://angularjs.org/');
    *
-   * @param {string} destination Destination URL.
+   * @param {string=} opt_destination Destination URL. Uses 'config.baseUrl' by default
```

Comment:
We're on TypeScript now, please remove typings from @param. Leaving the description is fine, but only if it's meaningful. ("@param opt_destination Destination URL" is not a meaningful comment, but mentioning the default is)